test/multi_dim_bc_test.jl

```diff
@@ -10,14 +10,14 @@ A = rand(n,m)
 
 #Create atomic BC
 q1 = RobinBC((1.0, 2.0, 3.0), (0.0, -1.0, 2.0), 0.1, 4.0)
-q2 = PeriodicBC(Float64)
+q2 = PeriodicBC{Float64}()
 
 BCx = vcat(fill(q1, div(m,2)), fill(q2, m-div(m,2)))  #The size of BCx has to be all size components *except* for x
 BCy = vcat(fill(q1, div(n,2)), fill(q2, n-div(n,2)))
 
 
-Qx = MultiDimBC{1}(BCx)
```

Comment:
This should be left as is to harmonise with the syntax for `CentredDifference`

Reply:
resolved by 61b0333, but see comment on line 91